R/data.table.R

```diff
@@ -1783,15 +1783,25 @@ replace_dot_alias = function(e) {
           is_constantish(q[["na.rm"]]) &&
             (is.null(q[["w"]]) || eval(call('is.numeric', q[["w"]]), envir=x))
         }
+        # run GForce for simple f(x) calls and f(x, na.rm = TRUE)-like calls where x is a column of .SD
+        .get_gcall = function(q) {
+          if (!is.call(q)) return(NULL)
+          # is.symbol() is for #1369, #1974 and #2949
+          if (!is.symbol(q[[2L]])) return(NULL)
+          q1 <- q[[1L]]
```

Comment:
```suggestion
          q1 = q[[1L]]
```

Reply:
thanks, since it's merged already I'll just leave it until we have the linter set up to enforce this globally